src/views/ActivitiesAll/index.js

```diff
@@ -58,10 +59,35 @@ export default class GordonActivitiesAll extends Component {
         activities,
         allActivities: activities,
         myInvolvements: myInvolvements,
-        loading: false,
+        loading: true,
```

Comment:
We probably don't need to update loading to true here since we already set that above. It only needs to be changed when we set it to false at the end of the method, right?